src/pages/en/editor-setup.md

```diff
@@ -11,7 +11,7 @@ Customize your code editor to improve the Astro developer experience and unlock
 
 ## VS Code
 
-[VS Code](https://code.visualstudio.com) is a popular code editor for web developers, built by Microsoft. The VS Code engine also powers popular in-browser code editors like [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io).
+[VS Code](https://code.visualstudio.com/) is a popular code editor for web developers, built by Microsoft. The VS Code engine also powers popular in-browser code editors like [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io/).
```

Comment:
Just checking that the codespaces link doesn't also need a trailing slash?

Reply:
When i go to the link it doesn't redirect me to `https://github.com/features/codespaces/` so it's doesn't need it